src/test/java/bio/terra/service/filedata/google/firestore/FileOperationTest.java

```diff
@@ -466,25 +469,34 @@ public void multiFileLoadSuccessTest() throws Exception {
     @Test
     public void multiFileLoadFailRetryTest() throws Exception {
         BulkLoadRequestModel loadRequest =
-            makeBulkFileLoad("multiFileLoadFailRetry", 0, 0, new boolean[]{true, false, true, false});
+            makeBulkFileLoad("multiFileLoadFailRetry", 0, 0, false, new boolean[]{true, false, true, false});
         loadRequest.maxFailedFileLoads(4);
         String loadTag = loadRequest.getLoadTag();
 
         BulkLoadResultModel result = connectedOperations.ingestBulkFileSuccess(datasetSummary.getId(), loadRequest);
         checkLoadSummary(result, loadTag, 4, 2, 2, 0);
 
         loadRequest =
-            makeBulkFileLoad("multiFileLoadFailRetry", 0, 0, new boolean[]{true, true, true, true});
+            makeBulkFileLoad("multiFileLoadFailRetry", 0, 0, false, new boolean[]{true, true, true, true});
         loadRequest.loadTag(loadTag);
         result = connectedOperations.ingestBulkFileSuccess(datasetSummary.getId(), loadRequest);
         checkLoadSummary(result, loadTag, 4, 4, 0, 0);
     }
 
+    @Test
+    public void multiFileLoadSuccessExtraKeysTest() throws Exception {
+        BulkLoadRequestModel loadRequest =
+            makeBulkFileLoad("multiFileLoadSuccessExtraKeys", 0, 0, true, new boolean[]{true, true, true, true});
+
+        BulkLoadResultModel result = connectedOperations.ingestBulkFileSuccess(datasetSummary.getId(), loadRequest);
+        checkLoadSummary(result, loadRequest.getLoadTag(), 4, 4, 0, 0);
+    }
+
     @Test
     public void multiFileLoadBadLineTest() throws Exception {
         // part 1: test that we exit with the bad line error when we have fewer than the max
         BulkLoadRequestModel loadRequest =
-            makeBulkFileLoad("multiFileLoadBadLineSuccess", 0, 3, new boolean[]{true, false, true, false});
+            makeBulkFileLoad("multiFileLoadBadLineSuccess", 0, 3, false, new boolean[]{true, false, true, false});
```

Comment:
It looks like you're adding in new positional params here but this PR isn't changing the method signature of `makeBulkFileLoad`.. what am I missing?